lab_6_pipeline/pipeline.py

```diff
@@ -97,6 +180,9 @@ def to_conllu(self, article: Article) -> None:
         Args:
             article (Article): Article containing information to save
         """
+        with open(article.get_file_path(kind=ArtifactType.UDPIPE_CONLLU),
+                  'w', encoding='utf-8') as f:
+            self._analyzer.write(split_by_sentence(article.text), 'conllu')
```

Comment:
you need to use library-related API. split by sentence is not required. is this method pointed in docs?

Reply:
I reworked this place. Split_by_sentence is not pointed in docs, but I need it to not pass string to the analyze function